### PR TITLE
feat: transcation success

### PR DESCRIPTION
**Pull request by showzyl**

### CODE REVIEW

File: lib/components/TokenConvert/index.vue [hunk 1]
```diff
@@ -462,8 +462,8 @@ async function doDeposit() {
         },
         memo: '',
         signerData: {
-            accountNumber: Number(acc.account.account_number),
-            sequence: Number(acc.account.sequence),
+            accountNumber: acc.account.account_number,
+            sequence: acc.account.sequence,
```
Comment (liangping) [inline lib/components/TokenConvert/index.vue]: Why do you change these line?

File: lib/components/TokenConvert/tokens.ts [hunk 2]
```diff
@@ -9,6 +9,13 @@ export interface TokenConfig {
 }
 
 export const tokens : TokenConfig[]= [
+    {
+        denom: "uside",
+        symbol: "uside",
+        ibcDenom: "uside", // use base denom for native token
+        decimals: 6,
+        coinImageUrl: "",
+    },
```
Comment (liangping) [inline lib/components/TokenConvert/tokens.ts]: Please remove this config here.

File: lib/components/TxDialog/index.vue [hunk 3]
```diff
@@ -101,7 +101,8 @@ const feeAmount = ref(2000);
 const feeDenom = ref('');
 const gasInfo = ref(200000);
 const memo = ref('');
-const chainId = ref('cosmoshub-4');
+// const chainId = ref('cosmoshub-4');
+const chainId = ref('taproot-1');
```
Comment (liangping) [inline lib/components/TxDialog/index.vue]: please use `comsoshub-4` as default chain-id

File: lib/components/TxDialog/messages/Send.vue [hunk 4]
```diff
@@ -20,6 +20,7 @@ const amountDenom = ref('')
 
 const msgs = computed(() => {
     const convert = new TokenUnitConverter(props.metadata)
+    console.log(`props: `, props);
```
Comment (liangping) [inline lib/components/TxDialog/messages/Send.vue]: remove console.

File: lib/wallet/UniClient.ts [hunk 5]
```diff
@@ -34,6 +34,7 @@ export class UniClient {
         //     authInfoBytes: signed.authInfoBytes,
         //     signatures: [fromBase64(signature.signature)],
         // });
+        console.log(`transaction: `, transaction);
```
Comment (liangping) [inline lib/wallet/UniClient.ts]: remove console

**Review by liangping — CHANGES_REQUESTED** [review]

File: lib/components/TokenConvert/index.vue (hunk 1 above)
Comment (showzyl) [reply lib/components/TokenConvert/index.vue]: i remove Number function and add this line

**Pull request merged**